lighthouse-core/audits/time-to-firstbyte.js

```diff
@@ -0,0 +1,111 @@
+/**
+ * @license
+ * Copyright 2017 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const Audit = require('./audit');
+const Formatter = require('../report/formatter');
+const URL = require('../lib/url-shim');
+
+const TTFB_THRESHOLD = 200;
+const TTFB_THRESHOLD_BUFFER = 15;
+
+class TTFBMetric extends Audit {
+  /**
+   * @return {!AuditMeta}
+   */
+  static get meta() {
+    return {
+      category: 'Performance',
+      name: 'time-to-firstbyte',
+      description: 'Time To FirstByte',
+      helpText: 'Time To First Byte identifies the time at which your server sends a response.',
```

Comment:
Maybe a Leare more link to https://developers.google.com/web/tools/chrome-devtools/network-performance/issues